goolib/goospec_test.go

```diff
@@ -205,6 +206,38 @@ func TestBadCompare(t *testing.T) {
 	}
 }
 
+func TestComparePriorityVersion(t *testing.T) {
+	for _, tc := range []struct {
+		p1   int
+		v1   string
+		p2   int
+		v2   string
+		want int
+	}{
+		{500, "1.2.3@1", 500, "1.2.3@2", -1},
```

Comment:
Can you add a  short description to each test case and print that when "got != tc.want"

Reply:
Done.